DigitalEcosystem/utils/figures.py

```diff
@@ -172,3 +172,37 @@ def create_multi_parity_plot(ytrue, series_to_plot, markers, colors, labels, alp
     plt.ylabel("Actual Volume (Å^3 / Formula Unit)")
     plt.legend(prop={"size": 8})
     plt.show()
+
+
+def publication_parity_plot(train_y_true,
```

Comment:
`save_parity_plot_publication_quality` ?

Reply:
Sure, I can give it that name